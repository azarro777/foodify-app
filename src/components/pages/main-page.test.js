import React from 'react';
import MainPage from './main-page';
import { shallow } from 'enzyme';

describe('Testing <MainPage/> component', () => {
  const main = shallow(<MainPage />);
  it('MainPage have rendered correctly', () => {
    expect(main).toMatchSnapshot();
  });
});
