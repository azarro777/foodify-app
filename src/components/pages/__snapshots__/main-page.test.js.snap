// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <MainPage/> component MainPage have rendered correctly 1`] = `ShallowWrapper {}`;
